server/liveblog/prepopulate.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8; -*-
```

Comment:
It is better to add prepopulate related changes to liveblog fork of superdesk:
https://github.com/liveblog/superdesk